VideoLocker/src/main/java/org/edx/mobile/discussion/DiscussionThread.java

```diff
@@ -172,4 +172,8 @@ public boolean isRead() {
         return read;
     }
 
+    public void setCommentCount(int commentCount) {
```

Comment:
Since we're only using this method to increment the count, replacing this with a method that does that might be more convenient and focused.